nhaystack-rt/src/nhaystack/driver/history/BNHaystackHistoryExport.java

```diff
@@ -205,9 +205,9 @@ private HRef locatePoint(HClient client)
   {
     BHistoryId id = getHistoryId();
     StringBuilder pointFilter = new StringBuilder();
-    pointFilter.append("point and axStation==");
+    pointFilter.append("point and n4Station==");
```

Comment:
Whilst I know this is your new code to do the export, I think you should look to achieve backwards compatibility by including the axStation tag in the filter.